.github/workflows/pd-e2e-test.yaml

```
@@ -0,0 +1,56 @@
+# This workflow runs end to end tests in Protocol Designer
+
+name: 'PD E2E Tests'
+
+on:
+  pull_request:
+    paths:
+      - 'protocol-designer/**'
+      - 'shared-data/**'
+      - 'components/**'
+      - 'webpack-config/**'
+      - '.package.json'
+  push:
+    paths:
+      - 'protocol-designer/**'
+      - 'shared-data/**'
+      - 'components/**'
+      - 'webpack-config/**'
+      - './package.json'
```

If you're feeling pedantic, for consistency above:

```suggestion
      - 'package.json'
```